test/functional/scan/tests/test-scan-Exclusive.hpp

```diff
@@ -54,7 +54,7 @@ void ScanExclusiveTestImpl(int N,
   res.memcpy(work_in, host_in, sizeof(T) * N);
   res.wait();
 
-  RAJA::exclusive_scan<EXEC_POLICY>(RAJA::make_span(work_in, N),
+  RAJA::exclusive_scan<EXEC_POLICY>(RAJA::make_span(static_cast<const T*>(work_in), N),
```

Comment:
are we still testing the non-const case? or, do we need to?

Reply:
We should not need to as this case is more restrictive, but it would be easy to just cast one of the two scans.